srml/council/src/lib.rs

```diff
@@ -14,7 +14,221 @@
+//!
+//! As opposed to motions, proposals executed through the Democracy module have the
+//! root origin, which gives them the highest privilege.
+//!
+//! #### Council Seats (seats.rs)
+//!
+//! The selection of council seats is a circulating procedure, constantly performing approval voting to accept
+//! a new council member and remove those whose period has ended. From the perspective of the
+//! [council seats module](./seats/trait.Trait.html), each tally (round of voting) is divided into two time periods:
+//!   - **Voting period:** In which any stakeholder can vote for any of the council candidates.
+//!   - **Presentation period:** In which voting is no longer allowed and stakeholders can _present_ a candidate
+//!   and claim that a particular candidate has won a seat.
+//!
+//! A tally is scheduled to execute based on the number of desired and free seats in the council.
+//!
+//! Both operations have associated bonds and fees that need to be paid based on the complexity of the transaction.
+//! See [`set_approvals`](./seats/enum.Call.html#variant.set_approvals) and
+//! [`submit_candidacy`](./seats/enum.Call.html#variant.submit_candidacy) for more information.
+//!
+//! Upon the end of presentation period, the leaderboard is finalized and sorted based on the approval
+//! weight of the _presented_ candidates.
+//! Based on the configurations of the module, `N` top candidates in the leaderboard are immediately recognized
+//! as councillors (where `N` is `desired_seats - active_council.len()`) and `C` of the remaining are kept in
+//! the leaderboard as _carry_ for the next tally to compete again. Similarly, councillors whose term has ended
+//! are also removed.
+//!
+//! - **Vote index**: A counter pointing the the number of tally rounds already applied.
+//! - **Desired seats:** The number of seats on the council.
+//! - **Candidacy bond:** Bond required to be a candidate. The bond is returned to all candidates at the end
+//! of the tally if they have won the tally (e.g. have become a councillor).
+//! - **Leaderboard:** A list of candidates and their respective votes in an election, ordered low to high.
+//! - **Presentation:** The act of proposing a candidate for insertion into the leaderboard. Presenting is
+//! `O(|number_of_voters|)`, so the presenter must be slashable and will be slashed for duplicate or invalid
+//! presentations. Presentation is only allowed during the "presentation period," after voting has closed.
+//! - **Voting bond:** Bond required to be permitted to vote. Must be held because many voting operations affect
+//! storage. The bond is held to disincentivize abuse.
+//! - **Voting:** Process of inserting approvals for oneself into storage. Can be called by anyone given they submit
+//! an appropriate list of approvals. A bond is reserved from a voter until they retract or get reported.
+//! - **Inactive voter**: A voter whose approvals are now invalid
```

Comment:
```suggestion
//! - **Inactive voter**: A voter whose approvals are now invalid (e.g. pointing to candidates who no longer exist).
```

Reply:
Is this example the definition of an inactive voter? Off the top of my head, I don't think it is. I thought it was someone who hasn't voted in too long?

Reply:
This should clarify: 

```
let valid = !Self::approvals_of(&who).iter()
				.zip(Self::candidates().iter())
				.any(|(&appr, addr)|
					 appr &&
					 *addr != T::AccountId::default() &&
					 Self::candidate_reg_info(addr).map_or(false, |x| x.0 <= last_active)
				);
```

A voter is inactive if at least for one of the candidates that it votes for:
- approval is _yay_.
- That candidate still exists
- That candidate was registered before the voter set their approvals for the last time.

I think in this paragraph mentioning any of the details is too much.